packages/utxo-lib/src/bufferutils.ts

```diff
@@ -82,18 +84,20 @@ export function writeInt64LE(buffer: Buffer, value: number, offset: number) {
 }
 
 export function readVarInt(buffer: Buffer, offset: number) {
-    const result = varuint.decode(buffer, offset);
+    const { numberValue, bytes } = varuint.decode(buffer, offset);
```

Comment:
The old interface returned just the number, and stored `bytes` directly on the `readVarInt`. Basically, the imported function (shared instance) had state containing a result of each function call :see_no_evil: 
Now it's a pure function, much better  :+1: 

See lib code: [old](https://github.com/bitcoinjs/varuint-bitcoin/blob/a05cb2cd32be43f6cab477704d60574066e3bb48/index.js#L21), [new](https://github.com/bitcoinjs/varuint-bitcoin/blob/d0b5f695480177cb25df7090215989a258da8413/src/cjs/index.cjs#L109)